README.md

```diff
@@ -101,6 +101,7 @@ const styles = StyleSheet.create({
 | renderYep           | func          | renders Yep label                     |              | () => null    |
 | renderNope          | func          | renders Nope label                    |              | () => null    |
 | onSwipeStart        | func          | function to be called when a card swipe starts                  |    | () => {} |
+| onChangeDirection   | func          | function to be called when a card swipe in active status                  |    | () => {} |
```

Comment:
Rename pls to onSwipeChangeDirection, update description and argument for this method